src/lineAuthor/view/gutter/commitChoice.ts

```diff
@@ -30,14 +29,7 @@ export function chooseNewestCommit(
 }
 
 function isNewerThan(left: BlameCommit, right: BlameCommit): boolean {
```

Comment:
Are you sure, that we need to check the epochSeconds and not also consider the timezone?

It's been a long time, since I last wrote this. But I am certain, that I've tested it back then and I think I would have tested it here to get the correct results. I am not sure, whether the new code and the old code are equivalent.

Reply:
According to your documentation of `epochSeconds`, it's in UTC. So comparing these should be sufficient. I even think the previous one was incorrect, because you want to compare the actual timestamp and not depend on timezone.

Reply:
Okay, then maybe I was wrong back then. Thx for fixing!